Source/WebCore/dom/Document.cpp

```diff
@@ -6511,6 +6511,11 @@ Document& Document::topDocument() const
     return *document;
 }
 
+CheckedRef<ScriptRunner> Document::checkedScriptRunner()
```

Comment:
If we want to keep this inline, we could also move it to DocumentInlines.h.

Reply:
I don't think we need to keep it inline. It's not in a hot code path.